### PR TITLE
copy edits

### PR DESCRIPTION
**Pull request by adaaaam**
Please review:  https://deploy-preview-1270--sourcegraph.netlify.app/blog/

adding in edits based on lori's feedback

### CODE REVIEW

File: blogposts/2020/3.18/announcing-sourcegraph-3.18.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ We’re improving Sourcegraph universal code search continuously, to help you so
 
 -   [Getting notified about the health of Sourcegraph is even easier](https://about.sourcegraph.com/blog/sourcegraph-health-notification)
 
-    Alerts are now delivered with detailed information, including links to potential solutions. Additionally, we now monitor container health and over/under-provisioning in all deployment modes -- so you will even get alerts if Sourcegraph needs more resources or could do with less!
+    Alerts are now delivered with detailed information, including links to potential solutions. Additionally, we now monitor container health and over/under-provisioning in all deployment modes – so you will even get alerts if Sourcegraph needs more resources or could do with less!
```
Comment (beyang) [inline blogposts/2020/3.18/announcing-sourcegraph-3.18.md:31]:
```suggestion
    Alerts are now delivered with detailed information, including links to potential solutions. Additionally, we now monitor container health and over/under-provisioning in all deployment modes, so you will be alerted if Sourcegraph needs more resources or could do with less.
```

File: blogposts/2020/3.18/combo-post.md [hunk 2]
```diff
@@ -14,9 +14,7 @@ GitHub issue: [#11044](https://github.com/sourcegraph/sourcegraph/issues/11044)
 
 Developer: [Loïc Guychard](https://github.com/lguychard)
 
-When you’re getting started setting up a Sourcegraph instance, it can be confusing to start the process of adding a code host. You may not be sure that Sourcegraph
-will protect the privacy of your data because there is no communication about how the software will interact with your repositories. We’ve added a new language in
-Sourcegraph 3.18 that clarifies what information we access within the Sourcegraph UI so you can move forward with confidence.
+When you’re getting started setting up a Sourcegraph instance, it can be confusing to start the process of adding a code host. You may not be sure that Sourcegraph will protect the privacy of your data because there is no communication about how the software will interact with your repositories. We’ve added new language in Sourcegraph 3.18 that clarifies what information we access within the Sourcegraph UI so you can move forward with confidence.
```
Comment (beyang) [inline blogposts/2020/3.18/combo-post.md:17]:
```suggestion
We've added in-product documentation that clarifies what data Sourcegraph accesses from your code host to address confusion that first-time users had when adding their first repositories to Sourcegraph. Sourcegraph takes data privacy seriously and only uses authentication tokens to access the data it needs to index your repositories.
```

File: blogposts/2020/3.18/search-multiple-revisions.md [hunk 3]
```diff
@@ -16,12 +16,12 @@ Often, you need to understand the differences between code at different branches
...
 
-A search over all branches, using [@\*refs/heads/](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24%40*refs/heads/+CONTAINER_ID&patternType=literal&case=yes) would look like this
+A search over all branches using [@\*refs/heads/](https://sourcegraph.com/search?q=repo:%5Egithub.com/sourcegraph/sourcegraph%24%40*refs/heads/+CONTAINER_ID&patternType=literal&case=yes) would look like this:
 
-![search over all branches using @*refs/heads/](images/searchacrossrev2.png)
+![Search over all branches using @*refs/heads/](/blog/3.18-search-all-branches.png)
 
-Be aware that searching over all branches may be slow. The cost of searching a branch is the same cost as searching a repository. To speed this up, ensure that branches are indexed with our experimental [Indexed non-master branches” feature](add link).
+Be aware that searching over all branches may be slow. The cost of searching a branch is the same cost as searching a repository. To speed this up, ensure that branches are indexed with our experimental [indexed non-master branches” feature](https://about.sourcegraph.com/blog/indexed-non-master-branches).
```
Comment (lorihey) [inline blogposts/2020/3.18/search-multiple-revisions.md:27]: This link is broken, and remove the floating/end quotation mark after branches

Comment (lorihey) [inline blogposts/2020/3.18/search-multiple-revisions.md:27]:
```suggestion
Be aware that searching over all branches may be slow. The cost of searching a branch is the same cost as searching a repository. To speed this up, ensure that branches are indexed with our experimental [indexed non-master branches feature](/blog/indexed-non-master-branches).
```

**Pull request merged**